.github/agents/generate-plugin.agent.md

```
@@ -1,7 +1,10 @@
+# SCF-Driven Content Model
+
+All post types, taxonomies, and field groups are now output as individual JSON files in `scf-json/` and registered by Secure Custom Fields (SCF). No PHP registration code is generated for post types or taxonomies.
 ---
 name: "Plugin Generator Agent"
```

markdownlint-cli2 (0.20.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)